<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>readModelDialog</class>
 <widget class="QDialog" name="readModelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>readModelDialog</string>
  </property>
  <widget class="QPushButton" name="conform">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancle">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>378</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>模型属性</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="modelType">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>point3d</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>point4d</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bezierLine</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nurbusLine</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nurbusSurface</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nurbusVol</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>58</y>
     <width>60</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>文件路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filePath">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>64</y>
     <width>189</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectFile">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
